src/components/support/Support.jsx

```diff
@@ -11,9 +11,9 @@ const Support = () => {
     <div className="px-10 bg-biscuits-orange-300">
       <Title text="Support Us" food="croissant" color="gray" />
       <div className="">
-        <div className="flex justify-items-end gap-x-6 py-14 justify-evenly items-center">
+        <div className="md:flex justify-items-end gap-x-6 py-14 justify-evenly items-center">
```

Comment:
maybe change the breakpoints to xl rather than md. otherwise looks pretty good
![image](https://github.com/user-attachments/assets/68c13986-7d99-4849-b6fb-551e19ecd144)